File: core/src/org/rexcellentgames/burningknight/debug/PassableCommand.java
package org.rexcellentgames.burningknight.debug;

import org.rexcellentgames.burningknight.entity.level.Level;

public class PassableCommand extends ConsoleCommand {
	{
		name = "/pas";
		shortName = "/p";
	}

	@Override
	public void run(Console console, String[] args) {
		Level.RENDER_PASSABLE = !Level.RENDER_PASSABLE;
	}
}

File: core/src/org/rexcellentgames/burningknight/debug/PassableCommand.kt
package org.rexcellentgames.burningknight.debug

import org.rexcellentgames.burningknight.entity.level.Level

class PassableCommand : ConsoleCommand("/pas", "/p", "Some debug") {

	override fun run(console: Console, args: Array<String>) {
		Level.RENDER_PASSABLE = !Level.RENDER_PASSABLE
	}
}